Common/src/main/java/com/natamus/treeharvester/processing/SaplingProcessing.java

```diff
@@ -0,0 +1,68 @@
+package com.natamus.treeharvester.processing;
+
+import com.natamus.collective.functions.BlockPosFunctions;
+import com.natamus.treeharvester.config.ConfigHandler;
+import com.natamus.treeharvester.data.Variables;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import oshi.util.tuples.Triplet;
+
+import java.util.Date;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class SaplingProcessing {
+    public static void onSaplingItem(Level level, Player player, ItemStack itemStack, BlockPos leafPos) {
+        if (level.isClientSide) {
+            return;
+        }
+
+        if (!ConfigHandler.replaceSaplingOnTreeHarvest || !ConfigHandler.automaticallyPickupItems) {
+            return;
+        }
+
+        Block block = Block.byItem(itemStack.getItem());
+        BlockPos yZeroPlayerPos = player.getOnPos().atY(0);
+//        BlockPos yZeroItemPos = leafPos.atY(0);
```

Comment:
@ricksouth I wish to hear your input on this. Should it search based on the player position or the leaf position?